conanfile.txt

```diff
@@ -1,5 +0,0 @@
-[requires]
```

Comment:
I think we don't want to remove this, as this is referenced in the README, as the getting started.
Probably we want to put it inside an ``examples`` folder?

Reply:
Sure, I can restore the example files and put them in a separate folder. At the moment I just moved the reference in the README to the test fixtures folder.

Reply:
Yes, I think the readme instructions with an example that users can reproduce to test things is important, so I'd prefer to keep it. I'd say to leave these 2 files in place for this PR, and it can be improved in a separate PR, let this one be focused on the testing.